std/range/package.d

```diff
@@ -8269,6 +8269,22 @@ unittest // issue 14373
     assert(r.empty);
 }
 
+unittest // issue 14575
+{
+    struct R
+    {
+        Object front;
+        alias back = front;
+        bool empty = false;
+        void popFront() {empty = true;}
+        alias popBack = popFront;
+        @property R save() {return this;}
+    }
+    static assert(isBidirectionalRange!R);
+	R r;
+	auto rr = refRange(&r);
```

Comment:
How is having `front` as a member variable "abnormal"? I do that in my own code quite often.